src/subframe_decoding.jl

```diff
@@ -381,3 +388,195 @@ function create_data(
 end
 
 
+function create_data_sub1(
+    TLM_HOW_Data::TLM_HOW_Data_Struct,
+    subfr_1_data::Subframe_1_Data)
+    
+
+    data = GPSData(
+        TLM_HOW_Data.integrity_status_flag,
+        TLM_HOW_Data.TOW,
+        TLM_HOW_Data.alert_flag,
+        TLM_HOW_Data.anti_spoof_flag,
+
+        subfr_1_data.trans_week,
+        subfr_1_data.codeonl2,
+        subfr_1_data.ura,
+        subfr_1_data.svhealth,
+        subfr_1_data.IODC,
+        subfr_1_data.l2pcode,
+        subfr_1_data.T_GD,
+        subfr_1_data.t_oc,
+        subfr_1_data.a_f2,
+        subfr_1_data.a_f1,
+        subfr_1_data.a_f0,
+
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing
+    )
+    return data
+end
+
+function create_data_sub2(
```

Comment:
This would be a good constructor:
```julia
function GPSData(prev_data::GPSData, TLM_HOW_Data::TLM_HOW_Data_Struct, subfr_2_data::Subframe_2_Data)
# ....
end
```